packages/manager/apps/carrier-sip/src/routing.js

```diff
@@ -0,0 +1,62 @@
+export default /* @ngInject */ ($stateProvider) => {
+  $stateProvider.state('app', {
+    url: '/billingAccount/:billingAccount/carrierSip/:serviceName',
+    component: 'carrierSipDashboard',
+    resolve: {
+      billingAccount: /* @ngInject */ $transition$ => $transition$.params().billingAccount,
+      carrierSip: /* @ngInject */ (
+        billingAccount,
+        CarrierSipService,
+        serviceName,
+      ) => CarrierSipService.getCarrierSip(billingAccount, serviceName),
+      cdrsLink: /* @ngInject */ (
+        $state,
+        billingAccount,
+        serviceName,
+      ) => $state.href('app.cdrs', { billingAccount, serviceName }),
+      currentActiveLink: /* @ngInject */ ($transition$, $state) => () => $state
+        .href($state.current.name, $transition$.params()),
+      dashboardLink: /* @ngInject */ (
+        $state,
+        billingAccount,
+        serviceName,
+      ) => $state.href('app', { billingAccount, serviceName }),
+      endpointsLink: /* @ngInject */ (
+        $state,
+        billingAccount,
+        serviceName,
+      ) => $state.href('app.endpoints', { billingAccount, serviceName }),
+      serviceInfos: /* @ngInject */ (
+        CarrierSipService,
+        serviceName,
+      ) => CarrierSipService.getServiceInfos(serviceName),
+      serviceName: /* @ngInject */ $transition$ => $transition$.params().serviceName,
+      settings: /* @ngInject */ (
+        billingAccount,
+        CarrierSipService,
+        serviceName,
+      ) => CarrierSipService.getSettings(billingAccount, serviceName),
+    },
+  });
+
+  $stateProvider.state('app.endpoints', {
+    url: '/endpoints',
+    component: 'carrierSipEndpoints',
+    resolve: {
+      endpoints: /* @ngInject */ (
+        billingAccount,
+        CarrierSipService,
+        serviceName,
+      ) => CarrierSipService.getEndpoints(billingAccount, serviceName),
+    },
+  });
+
+  $stateProvider.state('app.cdrs', {
```

Comment:
State should not be declared in the same file. There should be a `endpoints.routing` and a `cdr.routing`

Reply:
It's done.